### ISSUE TITLE
The runtime file for the downstream fine-tuning task cannot be found。

### ISSUE THREAD

**Issue by yanqiangmiffy** [facebookresearch/ConvNeXt#10]
For examples  'semantic_segmentation' :

`tools/dist_train.sh <CONFIG_PATH> <NUM_GPUS> --work-dir <SAVE_PATH> --seed 0 --deterministic --options model.pretrained=<PRETRAIN_MODEL>`

What is the  content in  `dist_train.sh`?

**Comment by liuzhuang13** [comment]
Thanks for trying the codebase; please refer to https://github.com/facebookresearch/ConvNeXt/issues/5
